services/frontend-service/src/ui/App/index.test.tsx

```diff
@@ -0,0 +1,112 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+import { App } from './index';
+import { render } from '@testing-library/react';
+import { Spy } from 'spy4js';
+import { AzureAuthSub } from '../utils/AzureAuthProvider';
+import { Observable } from 'rxjs';
+import { PanicOverview, UpdateOverview } from '../utils/store';
+
+Spy.mockModule('../components/NavigationBar/NavigationBar', 'NavigationBar');
+Spy.mockModule('../components/TopAppBar/TopAppBar', 'TopAppBar');
+Spy.mockModule('../components/ReleaseDialog/ReleaseDialog', 'ReleaseDialog');
+Spy.mockModule('./PageRoutes', 'PageRoutes');
+Spy.mockModule('../components/snackbar/snackbar', 'Snackbar');
+Spy.mockModule('../utils/AzureAuthProvider', 'AzureAuthProvider');
+
+const mock_GetConfig = Spy('Config');
+const mock_StreamOverview = Spy('Overview');
+
+jest.mock('../utils/GrpcApi', () => ({
+    // useApi is a constant, so we mock it by mocking the module and spying on a getter method instead
+    get useApi() {
+        return {
+            configService: () => ({
+                GetConfig: () => mock_GetConfig(),
+            }),
+            overviewService: () => ({
+                StreamOverview: () => mock_StreamOverview(),
+            }),
+        };
+    },
+}));
+
+const getNode = (): JSX.Element => <App />;
+const getWrapper = () => render(getNode());
+
+describe('App uses the API', () => {
+    beforeAll(() => {
+        jest.useFakeTimers('modern');
+        jest.setSystemTime(new Date('2/1/22'));
```

Comment:
Is there a way to initialize the date with a more readible value? like `2022-01-02`?